weepickle/src/main/scala/com/rallyhealth/weepickle/v1/WeePickle.scala

```diff
@@ -1,12 +1,15 @@
 package com.rallyhealth.weepickle.v1
 
+import com.rallyhealth.weepickle.v1.core.ops.FromInputOpsImplicits
 import com.rallyhealth.weepickle.v1.core.{FromInput, Visitor}
 
 /**
   * Converters for default scala types.
   * Macros to generate converters for case classes.
   */
-object WeePickle extends LowPriorityImplicits {
+object WeePickle
+  extends LowPriorityImplicits
+  with FromInputOpsImplicits {
```

Comment:
In the future, I expect to add `FromOps`/`ToOps`/`VisitorOps` for `.tolerant`, etc. Grouping the extensions into a "high level extension ops" traits seems like it should help discoverability.

Reply:
I think this would be a good thing to add to the code, it explains your reasoning which is often hard to "reverse engineer" from the code.

Reply:
also `BufferedValueOps`? https://github.com/rallyhealth/weePickle/blob/c1315d05edacb8e3dd7cb56838bc9cbfa8bfaf2f/weejson/src/main/scala/com/rallyhealth/weejson/v1/BufferedValue.scala#L35